README.md

```diff
@@ -21,7 +21,13 @@ $ ./capture.sh
 
 When you want to make a break just stop the process using `ctrl-c` and restart when you're done.
 
-Default capture rate is one screenshot every 4 seconds.
+Default capture rate is one screenshot every 4 seconds. This can however be adjusted with the --period command line option.
```

Comment:
Nit: options format

```suggestion
Default capture rate is one screenshot every 4 seconds. This can however be adjusted with the `--period` command line option.
```